src/main/proto/th2_grpc_data_provider/data_provider_template.proto

```diff
@@ -172,31 +207,37 @@ message MatchRequest {
   repeated Filter filters = 3;
 }
 
+/* Event search request */
+
 message EventSearchRequest {
-  google.protobuf.Timestamp start_timestamp = 1;
-  google.protobuf.Timestamp end_timestamp = 2;
-  EventID parent_event = 3;
-  TimeRelation search_direction = 4;
-  EventID resume_from_id = 5;
-  google.protobuf.Int32Value result_count_limit = 6;
-  google.protobuf.BoolValue keep_open = 7;
-  google.protobuf.Int64Value limit_for_parent = 8;
-  google.protobuf.BoolValue metadata_only = 9;
-  google.protobuf.BoolValue attached_messages = 10;
-  repeated Filter filters = 11;
+  google.protobuf.Timestamp start_timestamp = 1; // unix timestamp - Sets the search starting point. One of the 'start_timestamp' or 'resume_from_id' must not be null
+  google.protobuf.Timestamp end_timestamp = 2; // unix timestamp - Sets the timestamp to which the search will be performed, starting with 'start_timestamp'. When 'search_direction' is previous, end_timestamp must be less then start_timestamp. Defaults to null (the search is carried out endlessly into the past or the future).
+  EventID parent_event = 3; // Will match events with the specified parent element.
+  TimeRelation search_direction = 4; // Sets the lookup direction. Can be used for pagination. Defaults to NEXT.
+  EventID resume_from_id = 5; // last event id. In order to continue the execution of an interrupted sse request, you need to send exactly the same request with an indication of the element ID, from which to resume data transfer. Defaults to null. One of the 'start_timestamp' or 'resume_from_id' must not be null
+  google.protobuf.Int32Value result_count_limit = 6; // Sets the maximum amount of events to return. Defaults to 100
+  google.protobuf.BoolValue keep_open = 7; // If the search has reached the current moment, is it necessary to wait further for the appearance of new data. Default false
+  google.protobuf.Int64Value limit_for_parent = 8; // How many children for each parent do we want to request. Default not limited
+  google.protobuf.BoolValue metadata_only = 9; //  Receive only metadata (true) or entire event (false) (without attachedMessageIds). Default true
+  google.protobuf.BoolValue attached_messages = 10; // If the metadataOnly is false additionally load attachedMessageIds. Default false

```

Comment:
```suggestion
  google.protobuf.BoolValue attached_messages = 10; // If the metadataOnly is false, it will additionally load attachedMessageIds. Set by default to false.

```